doc/source/conf.py

```diff
@@ -40,7 +40,7 @@
     'sphinx.ext.autosummary',
     'sphinx.ext.extlinks',
     'sphinx.ext.intersphinx',
-    'numpydoc',
```

Comment:
I'm curious, what was the motivation for this change?

Reply:
Ah yes. Sorry. I was unable to build the docs locally without modifying the config file. 

![screen shot 2018-04-03 at 1 02 17 pm](https://user-images.githubusercontent.com/6101444/38280111-4a11126e-373f-11e8-98bc-cec6c14416b0.png)

Reply:
See the `doc/requirements-docs.txt` file.  From the error above it looks like you need to `pip install numpydoc`.

If you revert this change I'll be happy to merge.  Moving off of numpydoc would be a larger conversation than the doc change below.